camelot/admin/action/list_action.py

```diff
@@ -1098,23 +1098,19 @@ def _get_state(self, model_context, filter_value):
         state.modes = modes = []
         if len(filter_value) is not None:
             state.notification = True
+        # TODO: only show clear filter mode if any filters are active
+        #if len(filter_value):
+        modes.extend([Mode('__clear', _('Clear filter'), icon=Icon('minus-circle'))])
         selected_mode_names = [op + '-' + field for field, (op, *_) in filter_value.items()]
         for name, filter_strategy in self.get_filter_strategies(model_context):
```

Comment:
Ik heb voorlopig de clear filter bovenaan gezet, en de lijst met filters gesorteerd op priority level primair, en verbose name secondair. De functionaliteit is er ook om filters voor een specifieke priority level op te vragen, om die zodoende te kunnen scheiden met een seperator of dergelijke in de toekomst. De clear filter er tussen zetten voelde wat raar aan...